api/k8s/v1/k8s.proto

```diff
@@ -1055,6 +1063,53 @@ message DescribeNamespaceResponse {
   Namespace namespace = 1 [ (clutch.api.v1.log) = false ];
 }
 
+// ObjectKind represent a persistent entity in the system
+// For now we only support the kind Pod, we can expand on this
+// list as we add support for different kinds.
+enum ObjectKind {
+  UNSPECIFIED = 0;
+  UNKNOWN = 1;
+  POD = 2;
+}
+
+// this represents a Kubernetes event: https://pkg.go.dev/k8s.io/api/core/v1#Event
+// for a given object
+message Event {
+  string name = 1;
+  string reason = 2;
+  string description = 3;
+
+  // these values represent info of the object
+  // this event is about
+  string cluster = 4;
+  string namespace = 5;
+  // name of the object i.e pod name, deployment name, etc
+  string involved_object = 6;
+  // kind of the object e.g pod, deployment, service, etc
+  ObjectKind kind = 7;
+}
+
+message ListEventsRequest {
+  option (clutch.api.v1.id).patterns = {
+    type_url : "clutch.k8s.v1.Namespace",
+    pattern : "{cluster}/{namespace}"
+  };
+
+  string clientset = 1 [ (validate.rules).string = {min_bytes : 1} ];
+  string cluster = 2 [ (validate.rules).string = {min_bytes : 1} ];
+  string namespace = 3 [ (validate.rules).string = {min_bytes : 1} ];
+  // name of the object i.e pod name, deployment name, etc
+  string object = 4 [ (validate.rules).string = {min_bytes : 1} ];
```

Comment:
nit. or `object_name`
```suggestion
  string name = 4 [ (validate.rules).string = {min_bytes : 1} ];
```

Reply:
we can't call this `name` since there is already a field called `name` which refers to the event name. Can change to object_name if needed

Reply:
Updated to `object_name`